examples/ccvm_boxqp_plot.py

```diff
@@ -50,20 +52,55 @@
     # Save metadata to file
     metadata_filepath = metadata_list.save_metadata_to_file(METADATA_DIR)
 
-    plot_fig, plot_ax = ccvmplotlib.plot_TTS(
+    # Customize machine_parameters
+    machine_parameters = {
+        "cpu_power": {20: 5.0, 30: 5.0, 40: 5.0, 50: 5.0, 60: 5.0, 70: 5.0}
+    }
+
+    # If PLOT_OUTPUT_DIR not exists, create the path
+    if not os.path.isdir(PLOT_OUTPUT_DIR):
+        os.makedirs(PLOT_OUTPUT_DIR)
+        print("Plot folder doesn't exist yet. Creating: ", PLOT_OUTPUT_DIR)
+
+    # Plotting TTS
+    # Customize machine time calculating function
+    def cpu_machine_func(matching_df: pd.DataFrame, **_: any) -> float:
+        return np.mean(matching_df["solve_time"].values)
+
+    tts_plot_fig, tts_plot_ax = ccvmplotlib.plot_TTS(
         metadata_filepath=metadata_filepath,
         problem="BoxQP",
-        TTS_type="wallclock",
+        machine_time_func=cpu_machine_func,
     )
 
-    ccvmplotlib.apply_default_tts_styling(plot_fig, plot_ax)  # apply default styling
+    ccvmplotlib.apply_default_tts_styling(
+        tts_plot_fig, tts_plot_ax
+    )  # apply default styling
     plt.show()  # show plot in a new window
 
-    # If PLOT_OUTPUT_DIR not exists, create the path
-    if not os.path.isdir(PLOT_OUTPUT_DIR):
-        os.makedirs(PLOT_OUTPUT_DIR)
-        print("Plot folder doesn't exist yet. Creating: ", PLOT_OUTPUT_DIR)
+    # Save to local
+    tts_plot_fig.savefig(PLOT_OUTPUT_DEST)
```

Comment:
This seems to be the figure that got saved in the TTS plot file, but it looks like ETS to me. Can you check if this happens on your side?

![DL-CCVM_TTS_cpu_plot](https://github.com/1QB-Information-Technologies/ccvm/assets/19154892/ef61a459-f0d2-4055-87f0-3ca2c8058b5e)

Reply:
I am getting the same plots.
![ets](https://github.com/1QB-Information-Technologies/ccvm/assets/77126963/8e4049f3-9ad2-482b-a7f8-e6c78d000f8a)
![tts](https://github.com/1QB-Information-Technologies/ccvm/assets/77126963/e734b30f-1ed9-436e-9ada-f4b3175f446c)

Reply:
Hmm, okay let me try again!

Reply:
Okay I think I found the issue: both plots get saved to the same place: PLOT_OUTPUT_DEST (which we don't modify):
```
ets_plot_fig.savefig(PLOT_OUTPUT_DEST)
```

I think we need a variable here or a second constant.

Reply:
I separated the ETS and TTS plot output destination. And here are the new plots:
![ets](https://github.com/1QB-Information-Technologies/ccvm/assets/77126963/dfa87550-00dd-4f96-a99f-b7545e299148)
![tts](https://github.com/1QB-Information-Technologies/ccvm/assets/77126963/a7c64e78-7cd8-497c-bc9a-4afeed8af713)